common/src/main/java/io/druid/timeline/UnionTimeLineLookup.java

```diff
@@ -29,7 +30,7 @@
 
   public UnionTimeLineLookup(Iterable<TimelineLookup<VersionType, ObjectType>> delegates)
   {
-    this.delegates = delegates;
+    this.delegates = Iterables.filter(delegates, Predicates.notNull());
```

Comment:
can you add a comment explaining under what circumstances we may see null values?